doc/tuning.rst

```diff
@@ -96,7 +96,7 @@ You might be able to get a little more performance by using
 selections) rather than using :meth:`~.DataSet.select` with
 ``scans='track'``.
 
-When using MVF v4 one can also pass a `select` parameter to :meth:`katdal.open`
+When using MVF v4 one can also pass a `preselect` parameter to :meth:`katdal.open`
```

Comment:
The commit message seems the wrong way around :-)

Reply:
You're right. Oh well.

Reply:
I wasn't sure if `->` could also mean "replaces" 😄